plugins/lightspeed-backend/src/handlers/chatHistory.ts

```diff
@@ -14,19 +14,38 @@ export async function saveHistory(
   conversation_id: string,
   role: string,
   message: string,
+  timestamp?: number,
 ): Promise<void> {
   let newMessage: BaseMessage;
   switch (role) {
     case Roles.AIRole: {
-      newMessage = new AIMessage(message);
+      newMessage = new AIMessage({
+        content: message,
+        response_metadata: {
+          created_at: timestamp || Date.now(),
+          role: Roles.AIRole,
```

Comment:
it's already been stored as an `AIMessage`. the role is redundant here. 
we can get the `AIMessage` or `HumanMessage` from the id, as shown in your screenshot